js/ui/appDisplay.js

```diff
@@ -1731,6 +1735,10 @@ const AppIcon = new Lang.Class({
 
     popupMenu: function() {
         this._removeMenuTimeout();
+
+        if (!this._showMenu)
+            return true;
```

Comment:
I wonder why this needs to return true because the return value seems not used at all, and specially curious because the similar method in the old shell would return false, however I can see the function returns false when it succeeds (confusing again) so I guess returning the opposite here is ok?